models/issues/issue_search.go

```diff
@@ -22,7 +22,7 @@ import (
 // IssuesOptions represents options of an issue.
 type IssuesOptions struct { //nolint
 	db.ListOptions
-	RepoID             int64 // overwrites RepoCond if not 0
+	RepoIDs            []int64 // overwrites RepoCond if not 0
```

Comment:
The comment seems outdated.
```suggestion
	RepoIDs            []int64
```

Reply:
changed.